.env

```diff
@@ -0,0 +1,4 @@
+PORT=8000
```

Comment:
We should pick a unique port for this project and then add it to [our ports document](https://canonical-webteam.github.io/practices/project-structure/ports.html). E.g. `8032`.

Reply:
Updated, and PR created for the ports document.